python/main.py

```diff
@@ -111,6 +111,11 @@ def add_item_to_json(new_item):
     with open(file_path, "w") as file:
         json.dump({"items": items_list}, file)
 
+# 商品の削除
+@app.get("/delete/{item_id}")

```

Comment:
Usually, for deleting single resource, DELETE is used. Also the path is the same as GET (=`/items/{item_id}`).
Sometimes, POST method is used for deletion by expressing delete in path.
GET is almost always used for getting data, and if you modify or delete resources by GET.


DELETE METHOD: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE 
Also, Safe section of GET METHOD is good to read. https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET

If you have time, the course referred in Step3 will help.
> [Udemy Business - REST WebAPI サービス 設計](https://mercari.udemy.com/course/rest-webapi-development/)